### DISCUSSION TITLE
Release 1.6.2

### DISCUSSION THREAD

**Discussion by vomikan** [moneymanagerex/moneymanagerex#5432] [General]
It seems like now is the right time for a release.

**Comment by whalley** [comment]
@vomikan Agreed, seems quite stable. Unless you want to package and release now I can pick up in next day or so.

**Comment by n-stein** [comment]
@gabriele-v suggested possibly bumping version up to at least 1.7.0 due to the major changes of #1477. I think it's something to consider. Unlike 1.6.0 and 1.6.1 where a bit of SQL could easily drop the DB back down to v7 to restore compatibility with prior versions, this change drops a table and many columns making the jump irreversible.

**Comment by gabriele-v** [comment]
Please consider also the point that I highlighted in #1477. I think a warning for GRM is required, probably also GRM example reports should be updated as well.

**Reply by n-stein** [reply]
GRM examples were already updated with the rest of the code changes, the warning is a good idea though in case any user reports are broken.

**Comment by tactilis** [comment]
> bumping version up to at least 1.7.0 due to the major changes of https://github.com/moneymanagerex/moneymanagerex/issues/1477. I think it's something to consider.

Yes, I think we should do that.

However, most users will not read any significance into the new version being 1.7.0 as opposed to 1.6.2. They will just upgrade as a matter of course when MMEX alerts them that there is a new version.

**Reply by whalley** [reply]
1.6.2 is in the wild. Agree, users will just update anyway. The change will only affect a small number of 'power' users who use GRM and we have highlighted in the release notes the issue and provided example SQL in the GRM templates that they can use to upgrade any reports they have developed.
